src/types.ts

```diff
@@ -46,8 +46,30 @@ export interface GraphNode extends GraphElementBaseAttributes {
 
   /**
    * Fill color for the node.
+   * @deprecated use theme.fill
    */
   fill?: string;
```

Comment:
Kinda have to keep this for node specific colors though we could extend it to support a callback.